exporters/trace/src/test/java/com/google/cloud/opentelemetry/trace/TraceTranslatorTest.java

```diff
@@ -0,0 +1,198 @@
+package com.google.cloud.opentelemetry.trace;
+
+import com.google.devtools.cloudtrace.v2.AttributeValue;
+import com.google.devtools.cloudtrace.v2.Span;
+import com.google.devtools.cloudtrace.v2.TruncatableString;
+import com.google.rpc.Status;
+import io.opentelemetry.common.Attributes;
+import io.opentelemetry.common.ReadableAttributes;
+import io.opentelemetry.sdk.trace.data.SpanData;
+import io.opentelemetry.trace.Span.Kind;
+import io.opentelemetry.trace.*;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import java.util.*;
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.Assert.*;
+
+@RunWith(JUnit4.class)
+public class TraceTranslatorTest {
+
+  @Test
+  public void toDisplayNameTest(){
+    String serverPrefixSpanName = "Recv. mySpanName";
+    String clientPrefixSpanName = "Sent. mySpanName";
+    String regularSpanName = "regularSpanName";
+    Kind serverSpanKind = Kind.SERVER;
+    Kind clientSpanKind = Kind.CLIENT;
+
+    assertEquals("Recv. mySpanName", TraceTranslator.toDisplayName(serverPrefixSpanName, serverSpanKind));
+    assertEquals("Sent. mySpanName", TraceTranslator.toDisplayName(clientPrefixSpanName, clientSpanKind));
+    assertEquals("Recv.regularSpanName", TraceTranslator.toDisplayName(regularSpanName, serverSpanKind));
+    assertEquals("Sent.regularSpanName", TraceTranslator.toDisplayName(regularSpanName, clientSpanKind));
+  }
+
+  @Test(expected = NullPointerException.class)
+  public void nullTruncatableStringProtoTest(){
+    TruncatableString nullTruncatable = TraceTranslator.toTruncatableStringProto(null);
+
+    assertNull(nullTruncatable.getValue());
+    assertEquals(0, nullTruncatable.getTruncatedByteCount());
+  }
+
+  @Test
+  public void toTruncatableStringProtoTest(){
+    String truncatableString = "myTruncatableString";
+    TruncatableString testTruncatable = TraceTranslator.toTruncatableStringProto(truncatableString);
+
+    assertEquals("myTruncatableString", testTruncatable.getValue());
+    assertEquals(0, testTruncatable.getTruncatedByteCount());
+  }
+
+  @Test
+  public void toTimestampProtoTest(){
+    long epochNanos = TimeUnit.SECONDS.toNanos(3001) + 255;
+    com.google.protobuf.Timestamp timestamp = TraceTranslator.toTimestampProto(epochNanos);
+
+    assertEquals(3001, timestamp.getSeconds());
+    assertEquals(255, timestamp.getNanos());
+  }
+
+  @Test
+  public void toAttributesProtoTest(){
+    ReadableAttributes attributes = Attributes.newBuilder()
+            .setAttribute("myKey", "myValue")
+            .setAttribute("http.status_code", true)
+            .setAttribute("anotherKey", 100)
+            .setAttribute("http.host", 3.14)
```

Comment:
Although totally allowed often a nice side benefit to unit tests is they can show how the code should be used. To that end I might encourage keys and values match best practices. Namely we do try to interpret http.status_code and if we get a boolean we will be confused. You could just use "boolKey"... "intKey" etc.

Reply:
Added variables (stringKey, boolKey, longKey, doubleKey) and then put those variables into the parameters of `setAttribute` instead